contour/src/main/kotlin/com/squareup/contour/ContourLayout.kt

```diff
@@ -160,13 +160,28 @@ open class ContourLayout(
   attrs: AttributeSet? = null
 ) : ViewGroup(context, attrs) {
 
+  /**
+   * Whether Contour should respect padding set on this layout as part of laying out its subviews.
+   * When set to `true`, padding will influence the alignment of subviews to the layout's inner edges
+   * When set to `false`, padding will be ignored in all layout calculations.
+   * @see ViewGroup.setPadding
+   */
+  @Deprecated("Migrate implementation to properly take padding into account or explicitly null it out")
+  var respectPadding: Boolean = true
```

Comment:
Just spitballing: will it be more useful to enable padding for all layouts in the project?

Reply:
I don't think so - the benefit here is to allow for a careful, baby-steps migration. Setting a global flag would require someone to migrate an entire project in a single change.

For example, in Cash, my planned approach is to flag all `ContourLayouts` as `respectPadding = false`, put up a work sheet containing all of them, and parallelize the work of migrating them all.

Reply:
sounds fair